pysat/tests/test_constellation.py

```diff
@@ -264,7 +256,8 @@ class TestDataMod:
     def setup(self):
         '''Runs before every method to create a clean testing setup.'''
```

Comment:
Need to change the single quotation marks to double quotation marks.

Reply:
fixed here and elsewhere